src/Fabulous.XamarinForms/Xamarin.Forms.Core.fs

```diff
@@ -457,6 +458,15 @@ type [<Struct>] TabbedPage<'msg> (attrs: AttributesBuilder) =
 [<Extension>]
 type ViewExtensions () =
```

Comment:
Could you add also all the other attributes you declared, please?
Or you can also remove them for the time being if you don't actually need them.

Reply:
I have added the missing extensions and remove the one Im not using.